### PR TITLE
quilt-bot: Update logging

### PR DESCRIPTION
**Pull request by kayousterhout**
This PR updates the quilt-bot logging to be more useful.

### CODE REVIEW

File: review.go [hunk 1]
```diff
@@ -57,6 +58,7 @@ func processPullRequest(client *github.Client, pr *github.PullRequest) {
 		return
 	}
 	if len(reviewers) > 0 {
+		log.Printf("PR %d has %d outstanding reviewers", len(reviewers))
```
Comment (kklin) [inline review.go]: This log is missing an argument

Comment (kayousterhout) [reply review.go]: Fixed and made sure to actually compile this time.

File: review.go [hunk 2]
```diff
@@ -57,6 +58,8 @@ func processPullRequest(client *github.Client, pr *github.PullRequest) {
 		return
 	}
 	if len(reviewers) > 0 {
+		log.Printf("PR %d has %d outstanding reviewers",
```
Comment (kklin) [inline review.go]: Also, missing a \n

Comment (kklin) [reply review.go]: (I think). That or the above log has an extra \n

File: review.go (hunk 1 above)
Comment (kklin) [reply review.go]: FWIW I don't think that would've been a compile time error since the args are based on the string

File: review.go (hunk 2 above)
Comment (kayousterhout) [reply review.go]: It was missing -- fixed

**Pull request merged**